internal/clients/kibana/slo.go

```diff
@@ -21,6 +21,9 @@ func GetSlo(ctx context.Context, apiClient *clients.ApiClient, id, spaceID strin
 	ctxWithAuth := apiClient.SetSloAuthContext(ctx)
 	req := client.GetSloOp(ctxWithAuth, "default", id).KbnXsrf("true")
 	sloRes, res, err := req.Execute()
+	if err != nil && res == nil {
```

Comment:
```suggestion
	if res == nil {
```

This will still panic if `res == nil && err == nil`. Granted that probably shouldn't be possible, but asserting that `err != nil` here seems a little pointless.